app/device_classes/device_definitions/base_device.py

```diff
@@ -114,13 +107,15 @@ def get_cmd_output_with_commas(self, command, activeSession):
         Each array row is separated by newline.
         """
         result = self.run_ssh_command(command, activeSession)
-        result = self.replace_double_spaces_commas(result)
-        return self.split_on_newline(result)
-
-    def replace_double_spaces_commas(self, x):
-        """Replace all double spaces in provided string with a single comma."""
-        return fn.replaceDoubleSpacesCommas(x)
+        return result.replace("  ", ",").splitlines()
```

Comment:
I don't think this is correct (my bad).

We could probably do something like result.split("  ").strip()